charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 2.35.4
+version: 2.35.5
```

Comment:
don't think this is really needed, but the CI checks wanted a version bump. Happy to take it back out if not necessary.